src/datasources/cache/cache.router.ts

```diff
@@ -31,6 +31,7 @@ export class CacheRouter {
   private static readonly TRANSFERS_KEY = 'transfers';
   private static readonly ZERION_BALANCES_KEY = 'zerion_balances';
   private static readonly ZERION_COLLECTIBLES_KEY = 'zerion_collectibles';
+  private static readonly ZERION_RATE_LIMIT_KEY = 'zerion_rate_limit';
```

Comment:
I can forsee us using this for relaying. Maybe we should rename the key to `rate_limit` and then the method accept a prefix, e.g. `"zerion"` in this case. What do you think?

Reply:
Hmm, that would imply we are using the same rate limit parameters for all the APIs we want to rate limit, wouldn't it? I think it is valuable to decide the throughput for each case.

Reply:
Yeah, good suggestion, added in [7f4a85b](https://github.com/safe-global/safe-client-gateway/pull/1228/commits/7f4a85b6d803176df8a0d3894921bdc636508988)